sections/appendix/models.tex

```diff
@@ -0,0 +1,56 @@
+\subsection{Extenstions to our models}\label{subsec:app_exten_models}
+Looking at the results, in \autoref{tab:metric_results}, the Author-Topic and Category-Topic model does not get very high score in Topic Coherence.
+Trying to improve these results, we have run the combination model, using the same setup as the Author-Category model which is explained in \autoref{subsec:combinations}, just exchanging one feature with the document feature.
+Therefore, we create two extra model, Author-Doc and Category-Doc.
+We run them using the same hyperparameters as all the other models and they get similar results to the standard \gls{lda} model.
+
+\begin{table}[h]
+	\centering
+	\caption{Results.}
+	\begin{tabular}{l|c|c|c}
+		Topic Model & Perplexity & \makecell{Topic \\ Coherence} & \makecell{Topic \\ Difference} \\
+		\midrule
+		\Acrlong{lda} & 6761.6 & 0.520 & 0.575 \\
+		Author-Topic Model & 10991.5 & 0.335 & 0.615 \\
+		Category-Topic Model & 11126.0 & 0.370 & 0.560 \\
+		\textbf{Author-Doc Model} & \textbf{888036.4} & \textbf{0.543} & \textbf{0.574} \\
+		\textbf{Category-Doc Model} & \textbf{896045.7} &\textbf{ 0.530} & \textbf{0.575} \\
+	\end{tabular}
+	\label{tab:_extra_metric_results}
+\end{table}
+
+As \autoref{tab:_extra_metric_results} shows, the perplexity score for the two models, are extremely high, since we are multiplying three vectors instead of two.

```

Comment:
```suggestion
As \autoref{tab:_extra_metric_results} shows, the perplexity scores for the two models are extremely high since we are multiplying three vectors instead of two.

```